source/modules/MEI.xml

```diff
@@ -473,8 +473,7 @@
     </content>
   </macroSpec>
   <macroSpec ident="data.COLORNAMES" module="MEI" type="dt">
-    <desc>List of SVG-recognized color keyword names described at
-      https://www.w3.org/TR/SVG/types.html#ColorKeywords.</desc>
+    <desc>List of named colors from CSS Color Module Level 4.</desc>
```

Comment:
This change actually doesn't belong here

Reply:
something got mixed up